tests/test-team/pages/template-mgmt-choose-page.ts

```diff
@@ -0,0 +1,61 @@
+import { Locator, type Page } from '@playwright/test';
+import { TemplateMgmtBasePage } from './template-mgmt-base-page';
+import { Session, TemplateType } from '../../../src/utils/types';
+
+export class TemplateMgmtChoosePage extends TemplateMgmtBasePage {
+  readonly chooseTemplateRadioGroup: Locator;
+
+  readonly fieldsetHeading: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.chooseTemplateRadioGroup = page.getByRole('radiogroup');
+    this.fieldsetHeading = page.locator('[class="nhsuk-fieldset__heading"]');
+  }
+
+  static emptySessionData: Session = {
+    id: '3d98b0c4-6666-0000-0000-95eb27590000',
+    templateType: 'UNKNOWN',
+    nhsAppTemplateName: ' ',
```

Comment:
Shouldn't this be an empty string rather than a space?

Reply:
Yes ... sorted.